ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -1197,6 +1197,34 @@ public void shouldSupportVariableSubstitution() {
     assertThat(results.get(0).getStatementText(), is(csasSubstituted));
   }
 
+  @Test
+  public void shouldSupportVariableSubstitutionWithInitialVariables() {
```

Comment:
nit: (I wasn't sure what "initial variables" meant until I looked at the other test for comparison)
```suggestion
  public void shouldSupportVariableSubstitutionWithVariablesInRequest() {
```